app/Models/Airline.php

```diff
@@ -11,11 +11,39 @@ class Airline extends Model
 {
     protected $guarded = ['id'];
 
-    public function flights(): HasMany {
+    public function flights(): HasMany
+    {
         return $this->hasMany(Flight::class);
     }
 
-    public function cities(): BelongsToMany {
+    public function cities(): BelongsToMany
+    {
         return $this->belongsToMany(City::class);
     }
+
+    public function scopeOrder(Builder $query, ?string $sortBy, bool $ascending): void
+    {
+        $allowedColumns = ['id', 'name'];
```

Comment:
You can add this validation in the Request, https://laravel.com/docs/10.x/validation#rule-in